src/realm/error_codes.cpp

```diff
@@ -393,6 +393,8 @@ static const constexpr MapElem string_to_error_code[] = {
     {"SyncConnectFailed", ErrorCodes::SyncConnectFailed},
     {"SyncConnectTimeout", ErrorCodes::SyncConnectTimeout},
     {"SyncInvalidSchemaChange", ErrorCodes::SyncInvalidSchemaChange},
+    {"SyncLocalClockBeforeEpoch", ErrorCodes::SyncLocalClockBeforeEpoch},
```

Comment:
what does this error even mean?

Reply:
It's an error I added. We use 2015-01-01 as epoch and count the milliseconds since then for the changesets' timestamp. This error is thrown when the local clock is set before that date.